test/e2e/specs/test.spec.js

```diff
@@ -1,33 +1,83 @@
-import { mnemonics } from '../test.utils';
+import { mnemonics, phrasesList, MINUTE_MS } from '../test.utils';
+
 describe('Vaults UI Test Cases', () => {
   context('Test commands', () => {
-    it('should set up wallet', () => {
-      // Using exports from the synthetic-chain lib instead of hardcoding mnemonics UNTIL https://github.com/Agoric/agoric-3-proposals/issues/154
-      cy.setupWallet({
-        secretWords: mnemonics.user1,
-        walletName: 'user1',
-      }).then(taskCompleted => {
-        expect(taskCompleted).to.be.true;
-      });
+    const networkPhrases = phrasesList[Cypress.env('AGORIC_NET') || 'local'];
+
+    it('should setup the wallet', () => {
+      if (networkPhrases.isLocal) {
+        cy.setupWallet({
+          secretWords: mnemonics.user1,
+          walletName: 'user1',
+        });
+      } else {
+        const walletAddress = {
+          value: null,
+        };
+        cy.setupWallet({
+          createNewWallet: true,
+          walletName: 'my created wallet',
+        });
+        cy.origin('https://wallet.agoric.app/', () => {
+          cy.visit('/wallet/');
+
+          cy.get('input[type="checkbox"]').click();
+          cy.contains('Proceed').click();
+        });
+        cy.acceptAccess();
+
+        cy.origin('https://wallet.agoric.app/', () => {
+          cy.visit('/wallet/');
+
+          cy.contains(/agoric.{39}/).spread(element => {
+            return element.innerHTML.match(/agoric.{39}/)[0];
+          });
+        }).then(address => {
+          walletAddress.value = address;
+        });
+        cy.origin(
```

Comment:
Are we trying to get wallet address in this snippet?
```
        cy.origin('https://wallet.agoric.app/', () => {
          cy.visit('/wallet/');

          cy.get('input[type="checkbox"]').click();
          cy.contains('Proceed').click();
        });
        cy.acceptAccess();

        cy.origin('https://wallet.agoric.app/', () => {
          cy.visit('/wallet/');

          cy.contains(/agoric.{39}/).spread(element => {
            return element.innerHTML.match(/agoric.{39}/)[0];
          });
        }).then(address => {
          walletAddress.value = address;
        });
        
 ```
 
 If yes, I believe we created a method in `@agoric/synpress` for the same purpose.

Reply:
this is for getting the wallet address _before_ keplr has connected to anything. this means we cannot use the function in `@agoric/synpress` for this purpose